api/analysis.go

```diff
@@ -1058,11 +1058,9 @@ func (h AnalysisHandler) RuleReports(ctx *gin.Context) {
 			Name:         m.Name,
 		}
 		resources = append(resources, r)
-		if m.Labels != nil {
-			_ = json.Unmarshal(m.Labels, &r.Labels)
-		}
-		if m.Links != nil {
-			_ = json.Unmarshal(m.Links, &r.Links)
+		r.Labels = m.Labels
```

Comment:
r.Links = []Link{} so it's not nil?

Reply:
The `Links` field is `omitempty` so seemed like it was okay to let it be nil. No problem changing it if you disagree.